ci/docker-compose.yml

```diff
@@ -54,6 +54,17 @@ services:
     volumes:
       - ..:/risingwave
 
+  ci-flamegraph-env:
+    image: public.ecr.aws/x5u3w5h6/rw-build-env:v20230407
+    # NOTE(kwannoel): This is used in order to permit
+    # syscalls for `nperf` (perf_event_open),
+    # so it can do CPU profiling.
+    # These options should NOT be used for other services.
+    privileged: true
+    userns_mode: host
```

Comment:
Anyone more familiar with these can take a look? This is required to do profiling in docker.

Reply:
Btw I choose to go with `Docker` rather than on barebones linux machine since it is much easier to get benchmark tooling setup. We can always migrate back to barebones linux.

Another reason was that the barebones linux machine took up more space (30GB) at startup than the docker container (9.2G), not sure why.

Reply:
Worried that it could be user-facing, so would like a second opinion.

Reply:
I think it should be fine for *ci purposes* since it just gives same access as a normal host user on the machine.